Documentation~/preferences.md

```diff
@@ -1,279 +1,109 @@
-# ProBuilder Preferences window
+# ProBuilder Preferences
 
-To define how you interact with ProBuilder, you can customize the user interface and how the tools work.
+To configure ProBuilder, go to **Edit** > **Preferences** (macOS: **Unity** > **Settings**) in the main menu. In the **Prefereces** window, select **ProBuilder** from the list on the left.
 
-![Preferences Window](images/preferences.png)
+## Developer Mode
 
+Set what an element rotates around. This setting has a different effect depending on the selection mode. [[Jon I tried very hard to figure this out and couldn't; do you know?]]
```

Comment:
I don't have this option. Is this option only enabled when Developer mode is enabled?

Reply:
That makes sense.